shared/fs/common/upload-button.js

```diff
@@ -21,15 +21,27 @@ const UploadButton = Kb.OverlayParentHOC(props => {
     return <Kb.Button small={true} onClick={props.openAndUpload('both')} label="Upload" style={props.style} />
   }
   if (isIOS) {
-    return <Kb.Icon type="iconfont-new" padding="tiny" onClick={props.pickAndUpload('mixed')} />
+    return (
+      <Kb.Icon
+        type="iconfont-upload"
```

Comment:
This icon super jumps out at me here - maybe because I'm used to only having this as an in-progress icon, or maybe because it's big and blue - but it feels like it's more important than the rest of the screen. Can it be black instead maybe? Or did this come from design and I should comment somewhere in zeplin along these lines?

Reply:
Not on zeplin but this indeed came from design; @cecileboucheron wdyt? Should it remain blue?